OrcExamples/distributed_algorithms/processor-graph-superposables.inc

```diff
@@ -0,0 +1,79 @@
+{------ Superposed computation
```

Comment:
Since you're doing minor style fixes: "{- processor-graph-superposables.inc -- Orc include file that ...."

(several instances)